src/main/java/ch/njol/skript/expressions/ExprMoonPhase.java

```diff
@@ -0,0 +1,63 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import io.papermc.paper.world.MoonPhase;
+import org.bukkit.World;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Moon Phase")
+@Description("The current moon phase of a world")
+@Examples({
+	"if moon phase of player's world is full moon:",
+		"\tsend \"today's a full moon!\""
```

Comment:
```suggestion
		"\tsend \"Watch for the wolves!\""
```
maybe this is more fun 🤣